stix_shifter/stix_translation/src/modules/bigfix/README.md

```diff
@@ -1,180 +1,337 @@
+"{\"auth\":{\"username\":\"xxxxx\",\"password\":\"xxxxxx\"}}"
+query
+"<BESAPI xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:noNamespaceSchemaLocation=\"BESAPI.xsd\"><ClientQuery><ApplicabilityRelevance>true</ApplicabilityRelevance><QueryText>(\"Local Address\", local address of it as string | \"n/a\",  \"Remote Address\", remote address of it as string | \"n/a\",  \"Local port\", local port of it | -1,  \"remote port\", remote port of it | -1,  \"Process name\", names of processes of it,  pid of process of it as string | \"n/a\",  \"sha256\", sha256 of image files of processes of it | \"n/a\",  \"sha1\", sha1 of image files of processes of it | \"n/a\",  \"md5\", md5 of image files of processes of it | \"n/a\",  pathname of image files of processes of it | \"n/a\",  ppid of process of it as string | \"n/a\",  (if (windows of operating system) then  user of processes of it as string | \"n/a\"  else name of user of processes of it as string | \"n/a\"),  size of image files of processes of it | 0,  (if (windows of operating system) then  (creation time of process of it | \"01 Jan 1970 00:00:00 +0000\" as time -  \"01 Jan 1970 00:00:00 +0000\" as time)/second else  (start time of process of it | \"01 Jan 1970 00:00:00 +0000\" as time -  \"01 Jan 1970 00:00:00 +0000\" as time)/second),  \"TCP\", tcp of it, \"UDP\", udp of it)  of sockets whose (((local address of it as string contains \"192\" as string OR remote address of it as string contains \"192\" as string)) AND (if (windows of operating system) then (creation time of process of it | \"01 Jan 1970 00:00:00 +0000\" as time is greater than or equal to \"10 Jan 2013 08:43:10 +0000\" as time AND creation time of process of it | \"01 Jan 1970 00:00:00 +0000\" as time is less than or equal to \"23 Oct 2019 10:43:10 +0000\" as time) else (start time of process of it | \"01 Jan 1970 00:00:00 +0000\" as time is greater than or equal to \"10 Jan 2013 08:43:10 +0000\" as time AND start time of process of it | \"01 Jan 1970 00:00:00 +0000\" as time is less than or equal to \"23 Oct 2019 10:43:10 +0000\" as time))) of network</QueryText><Target><CustomRelevance>true</CustomRelevance></Target></ClientQuery></BESAPI>"
 
 ```
-[{"computer_identity": "12369754-bigdata4545.canlab.ibm.com", "subQueryID": 1, "start_time": "1541424881", "type": "process", "process_name": "systemd", "process_id": "1", "sha256hash": "74c4ff75e3623e64e3d6620864b69ed1d75fa460e520b88ed234234fsdfsdsdfs", "sha1hash": "916933045c5c91ebcaa325e7f8302f3123123dfgf0000", "md5hash": "28a9beb86c4d4c31ba572805baaa777f", "file_path": "/file/path/systemd"}]
+### A search_id is returned which is further passed to STIX transmission result module 
+```
+{'search_id': '4191', 'success': True}
+```
+### Transmit get result
 ```
+transmit bigfix "{\"host\":\"xx.xx.xx.xx\", \"port\":\"xxxxx\", \"cert_verify\":\"False\"}" {\"auth\":{\"username\":\"hclapi\",\"password\":\"jnGfdEYY8T$e\"}} results 4191 <offset> <length> 
```

Comment:
please sanitize the username and password. put something random.

Reply:
Incorporated with latest Commit.